web/src/css_variables.ts

```diff
@@ -12,6 +12,7 @@ const ms = 320; // Mobile small
 
 // Breakpoints for middle column
 const mc = 849; // Middle column as wide as it appears after the `sm` breakpoint
+const rc = 1350;
```

Comment:
This seems like an extremely large viewport before we show the right sidebar (150px more than the current 1200px breakpoint, I believe), especially at font sizes below 18px. I noticed some real awkwardness especially in inbox view, but this also does not seem optimal in message views, either.